applications/tari_dan_app_utilities/src/template_manager/implementation/service.rs

```diff
@@ -41,31 +59,46 @@ use super::{
     downloader::{DownloadRequest, DownloadResult},
     TemplateManager,
 };
-use crate::template_manager::interface::{TemplateExecutable, TemplateManagerError, TemplateManagerRequest};
+use crate::template_manager::interface::{
+    SyncTemplatesResult,
+    Template,
+    TemplateExecutable,
+    TemplateManagerError,
+    TemplateManagerRequest,
+};
 
 const LOG_TARGET: &str = "tari::template_manager";
```

Comment:
This was incorrect previously, please update this to the correct LOG_TARGET (see log4rs_sample it logs `tari::dan`) - just noticed I'm not seeing logs from this service.
```suggestion
const LOG_TARGET: &str = "tari::dan::template_manager";
```